react-frontend/src/components/StyleComponents/htmlTags.js

```diff
@@ -177,23 +181,15 @@ export const NavBarLink = styled(Link).attrs({
   }
 `;
 
-export const NavBarLinkFirst = styled(NavBarLink).attrs({
-  className: 'navBarLinkFirst',
-})`
-  @media (min-width: 800px) {
-    padding-left: 0;
-  }
-`;
-
 export const NavBarUl = styled.ul.attrs({
   className: 'navBarUl',
 })`
   color: #fff;
   display: flex;
   flex-direction: column;
   list-style: none;
+  margin-left: -75px;
```

Comment:
This was set to zero in the refactor so that the NavBarContainer could control the left alignment of the page

Reply:
I don't understand where and how this was being used. But with my refactor (because this is the wild west and we are all gunslingers refactoring code as we want to) things have changed. This style component is not being used at all. I am reverting this change. Whoever wants to cleanup the code or reuse this component can feel free to do so. :)